server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -384,6 +389,8 @@ private CommandExecutor startCommandExecutor(
                     return null;
                 });
             }
+            storagePurgeService.start(exec);
+            logger.info("Started the storage purge service on the leader replica ..");
```

Comment:
`..` means something is in progress. In this case, you finished starting the purge service already, so there's no need to append `..`.

Reply:
Ah, thanks!